src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,19 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    completed,
```

Comment:
I think it a good idea to restructure 'title' and 'user' also

Reply:
🤔